modules/widgets/contact-info.php

```diff
@@ -414,7 +414,10 @@ function has_good_map( $instance ) {
 					),
 					'https://www.google.com/maps/embed/v1/place'
 				);
-				$response = wp_remote_get( esc_url_raw( $path ) );
+				$wp_remote_get_args = array(
+					'headers' => array( 'Referer' => site_url() )
```

Comment:
What do you think about using `home_url()` instead of `site_url()` here? It may be more appropriate since that's most likely what the user would input in their Google API settings.